app/views/pages/home.html.erb

```diff
@@ -1,2 +1,8 @@
-<h1>Pages#home</h1>
-<p>Find me in app/views/pages/home.html.erb</p>
+<div class="banner" style="background-image: linear-gradient(rgba(0,0,0,0.4),rgba(0,0,0,0.4)), url(https://images.unsplash.com/photo-1446057032654-9d8885db76c6?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=2000&q=80);">
```

Comment:
put picture on cloudinary and link it is better, i think

Reply:
I thought maybe this was because it was on cloudinary that the script didn't work

Reply:
the script should work even if the pic is on cloudinary tho i think :/